Sources/CoverageHighlighter/CoverageHighlighter.swift

```diff
@@ -11,6 +11,19 @@ protocol NamedCoverable: Coverable {
     var name: String { get }
 }
 
+public struct Filter {
+    struct FilterFile {
+        let nameSuffix: String
```

Comment:
Basically the suffix that the file must contain
and the prefix that the functions inside, ideally should be and array not just one type of functions